installer/scripts/omsadmin.sh

```diff
@@ -70,6 +70,12 @@ OMSCLOUD_ID=""
 UUID=""
 USER_ID=`id -u`
 
+DEFAULT_SYSLOG_PORT=25224
+DEFAULT_MONITOR_AGENT_PORT=25324
```

Comment:
I don't think we will use more than 100 syslog ports. However, even it overflows into the monitor agent port range, it's still fine because it will assign the port only when the port is not occupied.